### DISCUSSION TITLE
How do I split a quarto book by section into webpages rather than by chapter into webpages? Chapters are too long for a webpage.

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#6242] [Q&A]
### Description

Normally, a quarto book is separated by chapter into webpages. Or if your book is long enough by parts, then into chapters (both of which) can be webpages. How do I split a quarto book by section into webpages rather than by chapter into webpages? Chapters are too long for a webpage. My book is too short for parts.

**Comment by mcanouil** [comment]
You can't.
Or consider your "parts" as chapters and your "sections" as chapters.

To note, I do not get your issue, as you can have a TOC (by default on the right) to navigate within chapter through the sections, what's wrong with that?

**Reply by ghost** [reply]
I have a book I'm writing that has 2 chapters, each with 4 sections. Quarto creates the pdf (from latex) just fine. However,  each chapter is a separate webpage, and each chapter has 4 sections on it. The webpages are way too long, especially since MaxJax is rendering on the webpages. I have tried changing the structure from chapter/section to part/chapter to accommodate Quarto's inadequacies, but that leads to a pdf that has parts and chapters in it, which I do not want. 

Is there a way to configure the translation so that the pdf does not use parts/chapters but rather uses chapters/sections?

**Reply by mcanouil** [reply]
You could use Quarto project Profiles one for PDF and one for HTML in which you have two different structure and settings but using the same files.

**Reply by ghost** [reply]
Thank you for your discussion. I was able to use the part/chapter method built into Quarto, but I had a LOT of LaTeXing to do for the pdf. I needed to reformat the TOC, headings, headers, footers, and several other items so the pdf looks like it uses chapter/sections like a normal book.  Maybe Quarto can fix this in future editions?
